spyder/plugins/editor/utils/debugger.py

```diff
@@ -124,6 +124,8 @@ def clear_breakpoints(self):
         for data in self.editor.blockuserdata_list():
             data.breakpoint = False
             # data.breakpoint_condition = None  # not necessary, but logical
+        self.editor.sig_breakpoints_changed.emit()
```

Comment:
Please add a comment @impact27 of why is this now needed, or why it was missing before

Reply:
Done. I don't know why it was missing but as the flags were recomputed completely for every print event (!!), this didn't matter.

Reply:
Understood!